syncserver/src/web/handlers.rs

```diff
@@ -32,6 +37,15 @@ pub async fn get_collections(
     db_pool: DbTransactionPool,
     request: HttpRequest,
 ) -> Result<HttpResponse, ApiError> {
+    let _hashed_fxa_uid: String = meta.user_id.hashed_fxa_uid.clone();
```

Comment:
A bit of a nit: Since these are prelude values that we're fetching, can we add a comment noting why we're cloning values we're immediately discarding? 
(That or we could add a comment as well as the commented prelude code, but I know that some folk are not fans of commented code blocks.)

Reply:
Sure, I can add a comment but this will go away right away as these are prefaced with `_` so clippy doesn't get mad. I want everything in place so that we can pass these into the glean code.

Reply:
Yep, totally understandable. 
One thing that @pjenvey drilled into my thick head was to make sure that each commit can "stand alone". Basically, if you were to run away to join the circus, could we figure out what you were doing six months from now?

It's surprisingly effective.

Reply:
Absolutely, that's a good rule to live by. Done. Let me know if the comment looks good for the interim.